src/dvc_render/vega_templates.py

```diff
@@ -508,23 +501,46 @@ class SmoothLinearTemplate(Template):
                     },
                     "color": {"field": "rev", "type": "nominal"},
                 },
-                "layer": [
+                "transform": [
                     {
-                        "mark": {
-                            "type": "line",
-                            "point": True,
-                            "tooltip": {"content": "data"},
-                        }
-                    }
+                        "loess": Template.anchor("y"),
+                        "on": Template.anchor("x"),
+                        "groupby": ["rev", "filename::field"],
```

Comment:
I think we need also `filename` and `field` here as well (VS Code doesn't always add that compound field)